applications/plugins/CGALPlugin/src/CGALPlugin/MeshGenerationFromPolyhedron.inl

```diff
@@ -152,8 +152,8 @@ template <class C3t3,class Obj>
 void printStats(C3t3& c3t3, Obj* obj, const char* step = "")
 {
     int nb_in = countWellCentered(c3t3);
-    obj->sout << step << ":  number of tetra     = " << c3t3.number_of_cells() << obj->sendl;
-    obj->sout << step << ":  well-centered tetra = " << ((double)nb_in/(double)c3t3.number_of_cells())*100 << "%" << obj->sendl;
+    msg_info(obj) << step << ":  number of tetra     = " << c3t3.number_of_cells()
+                  << step << ":  well-centered tetra = " << ((double)nb_in/(double)c3t3.number_of_cells())*100 << "%";
```

Comment:
a space is missing between c3t3.number_of_cells() and step

Reply:
?

Reply:
```<< c3t3.number_of_cells() << obj->sendl;
    obj->sout << step << ":  well-centered tetra = " <<```
replaced by 
``` << c3t3.number_of_cells() << step << ":  well-centered tetra = " <<```
A line return has been removed so it will stick the info, except if ```step``` is a tabulation ?

Reply:
Wow, this is what I call make a review. Hat's off for @epernod

Reply:
Fixed: I think It is better now.